Classes/BITFeedbackComposeViewController.m

```diff
@@ -151,13 +151,7 @@ - (void)keyboardWasShown:(NSNotification*)aNotification {
   NSDictionary* info = [aNotification userInfo];
   CGSize kbSize = [[info objectForKey:UIKeyboardFrameBeginUserInfoKey] CGRectValue].size;
   
-  BOOL isPortraitOrientation = NO;
-  
-#if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_8_0
-  isPortraitOrientation = UIInterfaceOrientationIsPortrait([[UIApplication sharedApplication] statusBarOrientation]);
-#else
-  isPortraitOrientation = UIInterfaceOrientationIsPortrait(self.interfaceOrientation);
-#endif
+  BOOL isPortraitOrientation = isPortraitOrientation = UIInterfaceOrientationIsPortrait([[UIApplication sharedApplication] statusBarOrientation]);
```

Comment:
> isPortraitOrientation = isPortraitOrientation = 

Hm? 🤔

Reply:
😩

Reply:
https://github.com/bitstadium/HockeySDK-iOS/pull/464/files includes this.